Now it's a test failure
```
>   # Check if there's a non-empty array first
>   if the_data:
E   ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

--- a/eodatasets3/images.py
+++ b/eodatasets3/images.py
@@ -784,7 +784,7 @@ def read_valid_mask_and_value_range(
         if calculate_percentiles is not None:
             the_data = array[valid_data_mask]
             # Check if there's a non-empty array first
-            if the_data:
+            if the_data.any():
                 low, high = np.percentile(
                     the_data, calculate_percentiles, interpolation="nearest"
                 )

